src/Filesystem/F.php

```diff
@@ -728,6 +728,9 @@ public static function remove(string $file): bool
 		}
 
 		$file = realpath($file);
+		if (is_string($file) === false) {
```

Comment:
Not important. I just wonder why you prefer `is_string` check instead simply `$file ===  false`? About easy code reading/reviewing?

Reply:
At the moment it's the same. My thought was that if `realpath()` ever returns something else than `string|false` for whatever reason, it's safer. Because `F::unlink()` only supports strings.

Reply:
I know it's the same. I was just curious because `realpath()` doesn'treturn any value other than `string|false`.